webpack.config.js

```diff
@@ -30,7 +30,9 @@ module.exports = {
       },
       {
         test: /\.css$/i,
-        use: ["style-loader", "css-loader"],
+        use: [
+          {loader: "style-loader"},
+          {loader: "css-loader", options: {url: false}}],
```

Comment:
install this? `https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode`